code/datums/ruins/icemoon.dm

```diff
@@ -115,3 +115,9 @@
 	Seem very intent on research and individual liberty, and also geology-based naming?"
 	prefix = "_maps/RandomRuins/AnywhereRuins/"
 	suffix = "golem_ship.dmm"
+
+/datum/map_template/ruin/icemoon/underground/syndicate_outpost //tegu edit, nuke ruin
```

Comment:
Please, put it in ModularTegustation and remove it from there. We'd like to avoid merge conflicts later on.

Reply:
The map file itself or this?